hydromt/flw.py

```diff
@@ -89,10 +89,13 @@ def flwdir_from_da(
 
 def d8_from_dem(
     da_elv: xr.DataArray,
-    gdf_stream: Optional[gpd.GeoDataFrame] = None,
     max_depth: float = -1.0,
-    outlets: str = "edge",
+    outlets: Literal["edge", "min"] = "edge",
```

Comment:
```suggestion
    outlets: Literal["edge", "min", "idxs_pit"] = "edge",
```
In pyflwdir, the docs for outlets is still "edge" or "min" and not "idxs_pit". Seems in the code though only "min" is checked so I guess in a way this works